### PR TITLE
added missing amq, pgsql, redis and sockets for PHP 7.3

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: 7.3/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ RUN apt-get update && apt-get install -y \
   libxpm-dev \
   libxml2-dev \
   libxslt-dev \
+  librabbitmq-dev \
```
Comment (OskarStark) [inline 7.3/Dockerfile]: this was missing, and theerfore the error message "pecl amqp..... error" occured

File: 7.3/Dockerfile [hunk 2]
```diff
@@ -38,13 +43,17 @@ RUN docker-php-ext-install pdo_mysql
 RUN docker-php-ext-install opcache
 RUN docker-php-ext-install bcmath
 RUN docker-php-ext-install calendar
+RUN docker-php-ext-install sockets
+
+RUN pecl install redis \
+    && docker-php-ext-enable redis
```
Comment (OskarStark) [inline 7.3/Dockerfile]: this was also missing in the 7.3 Dockerfile

**Comment by OskarStark** [comment]
local build was successfull at least for the 7.3 Dockerfile

**Comment by DracoBlue** [comment]
Thx!

**Pull request merged**